simulate.py

```diff
@@ -0,0 +1,41 @@
+from __future__ import (absolute_import, division,
+                        print_function, unicode_literals)
+
+import numpy as np
+import scipy as sp
+from scipy import signal
+import h5py
+import math
+import PSS_utils as utils
+import sys
+sys.path.append('/Users/nearclouding/Desktop/UTGRV_REU/nanograv_group')
+import VersionZeroPointZero as PSS
+
+#Initializing the signal
+S1=PSS.signal(Nt=2000)
+
+class Simulate():
+    """Class that simulates the pulsar signal
+       Insert more information about the class
+    """
+    def __init__(self, signal_class,pulsar_class,ism_class):
+        self.signal_class=signal_class
+        self.pulsar_class=pulsar_class
+        self.ism_class=ism_class
+        #NOTE need to give signal_class, pulsar_class, ism_class attributes?
+        #What else goes in here?
+
```

Comment:
Also, I've been thinking of a way to only enter the signal instance, `signal_class` and then just call the needed methods from the other packages, like `Pulsar()` and `ISM()`. 
> from . import ism.ISM
> self.DM = signal_class.MetaData.DM
> disperse()